File: app/src/main/java/com/m7mdabaza/fooddelivery/ui/activities/LoginActivity.java
package com.m7mdabaza.fooddelivery.ui.activities;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.Button;
import android.content.Intent;

import com.m7mdabaza.fooddelivery.R;
import com.m7mdabaza.fooddelivery.onBoarding.activities.OnBoardingActivity;

public class LoginActivity extends AppCompatActivity {

    Button logIn_btn;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_login);

        logIn_btn = findViewById(R.id.logIn_btn);

        logIn_btn.setOnClickListener(view -> {
            Intent intent = new Intent(LoginActivity.this, OnBoardingActivity.class);
            startActivity(intent);
        });
    }
}

File: app/src/main/java/com/m7mdabaza/fooddelivery/ui/activities/LoginActivity.kt
package com.m7mdabaza.fooddelivery.ui.activities

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.m7mdabaza.fooddelivery.R
import com.m7mdabaza.fooddelivery.onBoarding.activities.OnBoardingActivity
import kotlinx.android.synthetic.main.activity_login.*

class LoginActivity : AppCompatActivity() {



    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_login)

        login.setOnClickListener {
            val intent = Intent(this@LoginActivity, OnBoardingActivity::class.java)
            startActivity(intent)
        }

    }
}